tasks/packagemgmt.yml

```diff
@@ -212,16 +212,4 @@
   tags:
     - dnf
     - yum
-
-- name: python-rpm installation
```

Comment:
have you tested CentOS without this package?

Reply:
The python2-rpm installation is already inside the **pre.yml (line 96)** tasks, with correct check on the python version. This occurrence is duplicit and install python2 package on any system, not only system with python2 (depreceted).

Reply:
ah, you're correct.

Reply:
I'm going to cherry pick this removal, since it a unnecessary duplicate and this PR requires RHEL which I don't have at the moment.